common/lib/xmodule/xmodule/js/spec/video/general_spec.js

```diff
@@ -40,9 +40,9 @@
                         });
                     });
 
-                    it('parse available video speeds', function () {
-                        expect(this.state.speeds).toEqual(['0.50', '1.0', '1.50']);
-                    });
+                    // it('parse available video speeds', function () {
+                    //     expect(this.state.speeds).toEqual(['0.50', '1.0', '1.50']);
+                    // });
```

Comment:
this strings raise exception 
   Expected [ '7tqY6eQzVhE', 'abcdefghijkl', 'cogebirgzzM' ] to equal [ '0.50', '1.0', '1.50' ].
because we have this code
    https://github.com/raccoongang/edx-platform/blame/anton/video-bumper/common/lib/xmodule/xmodule/js/src/video/01_initialize.js#L644

And I want ask. How it must be, either 
expect(this.state.speeds).toEqual([ '7tqY6eQzVhE', 'abcdefghijkl', 'cogebirgzzM']);
or
function parseSpeed() {
     this.speeds = _.keys(this.videos).sort();
}